### DISCUSSION TITLE
METviewer line insert limit

### DISCUSSION THREAD

**Discussion by lindsayrblank** [dtcenter/METplus#1268] [Plot Generation] [answered]
Hi all!

I was wondering what the reasoning is behind the line insertion limit of 80000 per file? I'd like to insert files with more lines than that without needing to split them up, but do not want to break anything by doing so. 

Thanks!

**Comment by JohnHalleyGotway** [comment]
@TatianaBurek and/or @venitahagerty can you please take a look at this discussion and provide reasoning for the upper limit?

**Comment by TatianaBurek** [comment, marked as answer]
Hi Lindsay
The file size limit exists to make the application more secure and avoid any memory overflow problems.
So far all the MET output files that the users produced and loaded into the database were less then 80000 lines. 
Do you think it would be possible to decrease the size of your files by decreasing the number of statistics?
For example, do you need all the statistics from these files in the database or some of them could be ignored and not added to the MET output by changing MET configs.

**Reply by lindsayrblank** [reply]
Hi Tatiana,

Thank you for letting me know the reasoning. Unfortunately, I am unable to reduce the number of lines for these files, so I will just continue to split the files. Thank you anyway!
